workloads/kube-burner/workloads/node-pod-density/node-pod-density.yml

```diff
@@ -51,28 +50,25 @@ global:
         cert: {{ .CERTIFICATE }}
         key: {{ .PRIVATE_KEY }}
         url: https://localhost:2379/debug/pprof/profile?timeout=30
-
-
 {{ end }}
-
 jobs:
   - name: node-density
-    jobIterations: {{.JOB_ITERATIONS}}
-    qps: {{.QPS}}
-    burst: {{.BURST}}
+    jobIterations: ${JOB_ITERATIONS}
```

Comment:
shouldn't this be TEST_JOB_ITERATIONS? node-pod-density.yml is used by both pod and node density light versions but node density uses JOB_ITERATIONS but pod density uses TEST_JOB_ITERATIONS, this might lead to confusion . can we just use either one ?